xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-api/src/test/java/org/xwiki/model/internal/reference/DefaultStringEntityReferenceSerializerTest.java

```diff
@@ -318,7 +318,7 @@ void serializePageReferencesParameters()
             resolver.resolve("wiki:page1;param1=value1;param2=value2/page2;en_US", EntityType.PAGE);
         assertEquals("wiki:page1;param1=value1;param2=value2/page2;en_US", serializer.serialize(reference));
 
-        reference = resolver.resolve("wiki:pag\\;e1;pa\\;ram1=val\\;ue1;param2=value2/pag\\;e2;en_\\;US", EntityType.PAGE);
-        assertEquals("wiki:pag\\;e1;pa\\;ram1=val\\;ue1;param2=value2/pag\\;e2;en_\\;US", serializer.serialize(reference));
+        reference = resolver.resolve("wiki:pag\\;e1;pa\\;ram1=val\\;ue1;param2=value2/pag\\;e2;en_US", EntityType.PAGE);
+        assertEquals("wiki:pag\\;e1;pa\\;ram1=val\\;ue1;param2=value2/pag\\;e2;en_US", serializer.serialize(reference));
```

Comment:
Why did you change that test?

Reply:
I fixed the parsing to have the expected type (Locale) in the parameters, and "en_;US" is an illegal syntax for a locale.